feature-libs/cart/base/components/cart-coupon/cart-coupon.component.html

```diff
@@ -1,8 +1,16 @@
 <ng-container *ngIf="cart$ | async as cart">
   <div class="form-group">
+    <p class="form-legend">
```

Comment:
Best to add `cxFeatureLevel` to this and the `abbr` elements since they change rendering of the DOM.